babel.config.js

```diff
@@ -6,12 +6,10 @@ module.exports = function (api) {
   });
 
   const runtimePreset = isWeb
-    ? ['babel-preset-shopify/web', {modules: false, typescript: true}]
-    : ['babel-preset-shopify/node', {modules: 'commonjs', typescript: true}];
+    ? ['@shopify/babel-preset/web', {modules: false, typescript: true}]
+    : ['@shopify/babel-preset/node', {modules: 'commonjs', typescript: true}];
 
-  // babel-preset-shopify/react only uses HMR if hot is true and the env is
```

Comment:
The hot option was removed in https://github.com/Shopify/web-foundation/pull/145.
Storybook continues to auto reload without it.